src/transformStateWithClones.js

```diff
@@ -7,7 +7,42 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const clones = [];
+  let lastState = state;
+
+  for (const action of actions) {
+    let newState;
```

Comment:
Too much variables, here. You can work this out with only lastState variable

Reply:
you mean rewriting lastState every time ?

Reply:
Or just changing object 😉